doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
```

Comment:
In fact this is "from one of the the impacted subsystems".

Reply:
yes, I will change the language